code/controllers/configuration/entries/general.dm

```diff
@@ -64,6 +64,7 @@ Basics, the most important.
 /datum/config_entry/flag/allow_vote_mode    //allow votes to change mode
 
 /datum/config_entry/flag/allow_admin_jump   //allows admin jumping
+	config_entry_value = TRUE
```

Comment:
setting this default true because it was getting annoying, admins shouldn't be prevented from jumping